dangerfile.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const {markdown, danger} = require('danger');
+const fetch = require('node-fetch');
+
+const {generateResultsArray} = require('./scripts/rollup/stats');
+const currentBuildResults = require('./scripts/rollup/results.json');
```

Comment:
Could this somehow get cached by Node? Since we overwrite it, maybe we should `fs.readFileSync()` it?